charts/oes/config/oes-audit/audit-local.yaml

```diff
@@ -32,3 +32,6 @@ feign:
       url: http://oes-visibility:8096
     dashboard:
       url: http://oes-dashboard:8094
+  db:
+    migration:
+      enabled: true      
```

Comment:
@girichinna27  Why we are hardcoding as true ?? Is that not configurable ??

Reply:
@saitejaopsmx This specific configuration is only for this v4.0.3 / v4.0.3.x releases and we have db migration from v4.0.3 to v4.0.4, so I set it as true. If we want we can configure it from values.yaml